### PR TITLE
add psu input voltage and current

### PR DESCRIPTION
**Pull request by orfar1994**
Signed-off-by: orfar1994 <orfar1994@gmail.com>

Add PSU input voltage, input current and max power to the PSU table in the STATE DB

#### Description

psud would collect input voltage, input current and max power (capacity) of the PSU.
till now, it collect the output voltage and current.

#### Motivation and Context

more information about the PSU.

#### How Has This Been Tested?

unit tests, manual testing.

### CODE REVIEW

File: sonic-psud/tests/mock_platform.py [hunk 1]
```diff
@@ -282,7 +282,9 @@ def __init__(self,
                  temp_high_th=50.0,
                  voltage_low_th=11.0,
                  voltage_high_th=13.0,
-                 replaceable=True):
+                 replaceable=True,
+                 in_current=0.72,
```
Comment (assrinivasan) [inline sonic-psud/tests/mock_platform.py:286]: Just for my knowledge, from where did you get these values for V and I?

**Review by assrinivasan — APPROVED** [review]

File: sonic-psud/tests/mock_platform.py (hunk 1 above)
Comment (orfar1994) [reply sonic-psud/tests/mock_platform.py:286]: Do you mean the values 0.72 and 220.25?

Comment (assrinivasan) [reply sonic-psud/tests/mock_platform.py:286]: Yeah.

Comment (orfar1994) [reply sonic-psud/tests/mock_platform.py:286]: I took it from a real switch - 
for in_voltage - /sys/devices/platform/mlxplat/i2c_mlxcpld.2/i2c-2/i2c-5/5-0059/hwmon/hwmon6/in1_input
and for in_current - /sys/devices/platform/mlxplat/i2c_mlxcpld.2/i2c-2/i2c-5/5-0059/hwmon/hwmon6/curr1_input

**Review by stephenxs — APPROVED** [review]

**Pull request merged**